webpack.config.ts

```diff
@@ -43,6 +38,13 @@ interface Configuration extends webpack.Configuration {
 
 const {env} = process;
 
+if (env.EXTRACT_IOS_DEVICE_NAMES) {
```

Comment:
i think everyone building webpack needs this to have run for the file to exist

Reply:
oh its committed

Reply:
The file exists on Git right now: https://github.com/getsentry/sentry/blob/master/static/app/constants/ios-device-list.tsx